pkgs/top-level/all-packages.nix

```diff
@@ -27754,7 +27754,7 @@ with pkgs;
   };
 
   libreoffice = if stdenv.isDarwin
-    then callPackage ../applications/office/libreoffice/darwin.nix {}
+    then callPackage ../applications/office/libreoffice/darwin/darwin.nix {}
```

Comment:
```suggestion
    then callPackage ../applications/office/libreoffice/darwin { }
```

Reply:
@SuperSandro2000 This only works if rename `darwin.nix` -> `default.nix`. Did you mean that?

Reply:
yeah, I implied that

Reply:
done